model/vowpalWabbit/dffml_model_vowpalWabbit/vw_base.py

```diff
@@ -77,7 +70,7 @@ class VWConfig:
     )
     convert_to_vw: bool = field(
```

Comment:
Let's change the name so that it's a default of False, because when we give it on the command line a bool flag that has a default of False, to set it to true all we need to do is say `-flag` rather than `-flag false` or `-flag true`